api/test/search.test.js

```diff
@@ -17,7 +17,7 @@ describe('search', () => {
 
     const { metabolite } = data['Human-GEM'];
     expect(metabolite.length).toBeGreaterThan(0);
-    expect(metabolite.length).toBeLessThan(50);
+    expect(metabolite.length).toBeLessThan(51);
```

Comment:
If I remember correctly, @e0 made it by purpose to demonstrate a failed test, since I was suggesting using `toBeLessThanOrEqual(50)`. Maybe we don't need this failed testing example anymore?

Reply:
@e0 any comment on this?

Reply:
Looks like this has been changed to `toBeLessThanOrEqual(50)` in the new files, which seems very appropriate.